launch/launcher.go

```diff
@@ -142,9 +142,13 @@ func eachDir(dir string, fn func(path string) error) error {
 	return nil
 }
 
-func (l *Launcher) eachBuildpack(dir string, fn func(path string) error) error {
+func (l *Launcher) eachBuildpack(fn func(path string) error) error {
+	layersDir, err := filepath.Abs(l.LayersDir)
```

Comment:
Does this change relate to subject of this PR - "Remove all unsafe usage of filepath.Glob", or did you just do it on the way?
I think it's good that we're not passing the `dir` to `eachBuildpack` anymore, but I don't understand why we need `filepath.Abs`.

Reply:
I just made this refactor along the way. We were previous doing this for profile scripts here https://github.com/buildpacks/lifecycle/pull/432/files#diff-470373c3e39c585a84421cca78d2d178L62-L65. We weren't previously doing this for env files (the other comsumers of `eachBuildpack`), but it doesn't hurt and I like the consistent interface. If I call `eachBuildpack`, I don't have to worry about absolute vs relative paths.

Reply:
This is what I thought. Great! Thanks!

Reply:
After thinking more about your question I ended up moving this to make it profile script specific again to make this less confusing and avoid changing behavior unnecessarily.